catalog-info.yaml

```
@@ -0,0 +1,12 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component                                       # Alterar para 'Component', 'API' ou 'Resource'
+metadata:
+  name: searchops
+  description: None
+  annotations:
+    github.com/project-slug: gupy-io/searchops
+    backstage.io/techdocs-ref: dir:. 
+spec:
+  type: service                                       # Alterar para 'service', 'website' ou 'library'
+  owner: catalogo-incompleto                          # Confome nome no GitHub ('qualificacao', 'vagas', 'candidates', por exemplo)
```

```suggestion
  owner: engenharia                          # Confome nome no GitHub ('qualificacao', 'vagas', 'candidates', por exemplo)
```

Deve ficar com o time de busca&api mas ele ainda nao existe então é de todes